doc/reported_states_during_deployment.md

```diff
@@ -0,0 +1,375 @@
+.properties.reported.lastDesiredStatus.code == 200 && 
+```
+*and for any desired module*:   
+```
+   .properties.desired.modules.<module>.status  != .properties.reported.modules.<module>.runtimeStatus &&  
+   .properties.desired.modules.<module>.settings.image == .properties.reported.modules.<module>.settings.image 
+```
+
+NOTE: Modules may go into a failed state at any time for any number of reasons, it is up to the 
+user's discretion to determine the appropriate amount of time to wait to determine if a module has 
+not been successfully deployed. 
+
+### Deployment in progress/Deployment failed.
+
+One of the steps that the IoT Edge runtime attempted to apply the current desired state failed. 
+There may be many reasons for this, such as an image pull request is taking too long, the module 
+specification references a bad image, or the container runtime failed for any number of reasons.
+
+
+```
+.configurations.<deployment name>.Status: "Applied" && 
+.properties.desired."$version" == .properties.reported.lastDesiredVersion && 
+.properties.reported.lastDesiredStatus.code == 500 &&
+```
+*and for any desired module*:   
+```
+   .properties.desired.modules.<module>.settings.image != .properties.reported.modules.<module>.settings.image 
+```
+
+NOTE: if this is a new module in the deployment, it will not exist in the reported properties until 
+the module is created.
+
+NOTE: If the module was previously marked as “running” it will stay “running” using the previous 
+image until the pull for the new module is complete.
+
+NOTE: Some module images may be very large or download very slowly. The IoT Edge runtime will only 
+wait for just over a minute for this pull to complete before timing out. The pull will still be in 
+progress, but the runtime will report this attempt as unsuccessful until the pull is complete. The 
+IoT Edge runtime doesn’t have a clear indicator to distinguish between this state and a failed image 
+pull. You may be able to derive diagnostic information from `.properties.reported.lastDesiredStatus.description`. 
+For example, if the pull was timed out, the text will contain “The operation was canceled”, and this 
+state is probably transitional.
+
+# Changes for individual deployments
+
+This document focused on at-scale deployments, but the IoT Edge runtime behavior is identical for an 
+[individual device deployment](https://docs.microsoft.com/azure/iot-edge/how-to-deploy-modules-portal?view=iotedge-2020-11). 
+The primary difference will be the **absence** of a [Configuration section](#configuration-section) in \$edgeAgent twin.
+
+The Configuration Section is set by the service, but it is not read or interpreted by the IoT Edge 
+runtime. All reconciliation actions taken by the IoT Edge runtime are derived from the [Desired Properties section](#desired-properties-section). 
+The fields like `.configurations.<deployment name>.Status` will not be present for this device and may be 
```

Comment:
replace "will not be present for this device and may be ignored" with "will not be present for individual device deployments"